data/ulid.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @title ULID
+ * @difficulty beginner
+ * @tags cli, deploy
+ * @run <url>
+ * @resource {https://github.com/ulid/spec} ULID: Specification
+ *
+ * One common need for distributed systems are identifiers. ULIDs are a universally
+ * unique lexicographically sortable identifier with some nice properties. They are
+ * 128-bit values, encoded as 26 character strings which also encode the timestamp.
+ * They play very nicely with Deno KV.
+ */
+
+// The standard library contains a function for generating ULIDs.
+import { ulid } from "$std/ulid/mod.ts";
+
+// To generate a ULID, simply call the function.
+console.log(ulid());
+console.log(ulid());
+console.log(ulid());
```

Comment:
Nit: perhaps only one of these is needed.

Reply:
I think it's important to show that they're all different, but I don't feel strongly about this.